backend/models/alternative.py

```diff
@@ -0,0 +1,45 @@
+from .shared import db # Allows the models to be split out into separate files.
+from models.value import Value
+
+class Alternative(db.Model):
+    __tablename__ = 'alternative'
+    id = db.Column(db.Integer, primary_key=True)
+
+    hierarchy_id = db.Column(db.Integer, nullable=False)
+    name = db.Column(db.String())
+
+    
+    # Currently contains all nodes in the tree
+    values = db.relationship(
+        "Value",
+        cascade="all, delete",
+        backref=db.backref("alternative")
+        )
+
+    def __init__(self, hierarchy_id, name):
+        """
+        Parameters
+        ----------
+        name : str
+            Name of the alternative to be created.
+
+        """
+        self.name = name
+        self.hierarchy_id=hierarchy_id
+
+    def to_dict(self, get_values=True, export=False):
+        alt_dict = {
+            "name": self.name,
+            "hierarchy_id": self.hierarchy_id

```

Comment:
Make any values sent in an api response camelcase. So "hierarchyId" and no underscore